src/project-type/workspace-instance.ts

```diff
@@ -26,12 +22,12 @@ function initWorkspaceImpl(workspaceImpl: WorkspaceApi): void {
 
 function createWorkspaceReadOnlyProxy(workspaceImpl: WorkspaceApi): BasWorkspaceApi {
   const basWSAPI: BasWorkspaceApi = {
-    getProjects: (tag?: Tag): Promise<ProjectApi[]> => workspaceImpl.getProjects(tag),
-    
-    getProjectUris: (): Promise<Uri[]> => workspaceImpl.getProjectUris(),
-
-    onWorkspaceChanged: (handler: (event: string, folders: WorkspaceFolder[]) => void): void => 
-      workspaceImpl.onWorkspaceChanged(handler)
+    // eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
You can disable a whole block (start/stop) in ESLInt.
That would be more useful particularly if more methods would be added to the BasWorkspaceAPi